Assets/Scripts/Game/DaggerfallBankManager.cs

```diff
@@ -77,7 +77,7 @@ public enum ShipType
 
     public static class DaggerfallBankManager
     {
-        public const float goldUnitWeightInKg = 0.0025f;
+        public static float goldUnitWeightInKg = DaggerfallUnity.Instance.ItemHelper.GetItemTemplate(ItemGroups.Currency,0).baseWeight;
```

Comment:
The base weight of gold is set as 0.5kg in item templates so this is quite a change, gold pieces will start weighing 200 times as much as it should if this PR is merged as-is.

Reply:
ItemGroups.Currency,0 returns item template index 276, which is Gold Pieces, which has a baseWeight of 0.0025
AFAIK, Gold Pieces is the only element in ItemGroups.Currency
Even if not, I tested this change with a mod I made, and it returns the baseWeight from index 276.

Reply:
You're correct, I found the 'Gold' entry which is for a bar of gold. Nevermind, thanks for checking.